test/test_jit.py

```diff
@@ -4400,6 +4446,32 @@ def forward(self, x, seq_lens):
             f = io.BytesIO()
             torch.onnx._export(m, (x, seq_lens), f, verbose=False)
 
+    def test_python_call_non_tensor(self):
+        def foo(a, b, c):
+            d, e = c
+            # type: (Tensor, int, Tuple[Tensor, int]) -> Tuple[int, Tensor]
```

Comment:
Doesn't this need to be on the first line?

Reply:
I checked and it doesn't actually matter. I think the regex doesn't actually check that it has to be the first line. I changed the test regardless.